app/models/patient.rb

```diff
@@ -520,7 +520,7 @@ def latest_transfer
       .where(isolation: true)
       .where(symptom_onset: nil)
       .where.not(latest_assessment_at: nil)
-      .where('first_positive_lab_at < ?', ADMIN_OPTIONS['recovery_period_days'].days.ago)
+      .where('first_positive_lab_at < ?', ADMIN_OPTIONS['asymp_non_test_based_recovery_period_days'].days.ago)
```

Comment:
I think this could go either way, but just wanted to bring it up as a thought: Would it make sense for these scopes to return no patients if `ADMIN_OPTIONS['enable_asymp_non_test_based']` is set to `false`? I know the recovery definition works fine the way it is now but I wonder if it would make more sense in terms of consistency to just have this scope not return any patients if it's disabled or if that would be confusing

Reply:
I did actually go back and forth on this a little when implementing. Kind of a toss up but I'm good with doing it this way https://github.com/SaraAlert/SaraAlert/pull/1238/commits/3f335035788242a1914e725c39f42d672516af32